superdesk/core/resources/model.py

```diff
@@ -65,7 +66,7 @@ def type(self) -> str:
         return self.model_resource_name
 
     #: ID of the document
-    id: Annotated[Union[str, ObjectId], Field(alias="_id")]
+    id: Annotated[Union[str, ObjectId], Field(alias="_id", default_factory=lambda: generate_guid(type=GUID_NEWSML))]
```

Comment:
Now I'm thinking we should also use `default_factory` for `created` and `updated` too. It would more clear that they are automatically set if not provided, therefore we don't need a service class do it for them. What do you think?

Reply:
I started to make this change. But, using `default_factory` here doesn't work because when you update the item, the `_updated` field already has a value and therefor does not get updated.

It also makes sense to set these values in the service, right before it is sent to the DB. So the `_created` and `_updated` fields are the date/time of when it was stored at rest (aka the DB)